packages/app/src/components/time-series-chart/components/tooltip/tooltip-series-list-items.tsx

```diff
@@ -180,6 +180,7 @@ const TooltipEntryValue = styled.span<{
 }>((x) =>
   css({
     textAlign: x.isCentered ? 'center' : 'right',
+    width: x.isCentered ? '100%' : 'unset',
```

Comment:
is the unset necessary?
```suggestion
    width: x.isCentered ? '100%' : undefined,
```